### PR TITLE
feat: Show manual backport instructions

### PR DESCRIPTION
**Pull request by ChristophWurst**
This is hopefully helpful for everyone new to the concept of "backporting manually".

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Comment by ChristophWurst** [comment]
> Some checks were not successful

Now what? :fish: :fish_cake: :tropical_fish:

**Comment by skjnldsv** [comment]
```
#3 ERROR: failed to do request: Head "https://registry-1.docker.io/v2/nextcloudci/node/manifests/node-7": net/http: TLS handshake timeout
------
 > [internal] load metadata for docker.io/nextcloudci/node:node-7:
------
Dockerfile:1
--------------------
   1 | >>> FROM nextcloudci/node:node-7
   2 |     
   3 |     RUN apk add python
--------------------
ERROR: failed to solve: nextcloudci/node:node-7: failed to do request: Head "https://registry-1.docker.io/v2/nextcloudci/node/manifests/node-7": net/http: TLS handshake timeout
Error: buildx failed with: ERROR: failed to solve: nextcloudci/node:node-7: failed to do request: Head "https://registry-1.docker.io/v2/nextcloudci/node/manifests/node-7": net/http: TLS handshake timeout

```

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/backportbot/blob/d00ce084c0309736591bb77162f6938b7a04c310/.github/workflows/docker-publish.yml#L15-L16

Maybe it's not such a good idea :thinking:

File: lib/pr.js [hunk 1]
```diff
@@ -115,7 +115,25 @@ module.exports = {
...
-        body: 'The backport to ' + target.branch + ' failed. Please do this backport manually.'
+        body: `The backport to ${target.branch} failed. Please do this backport manually.
+
+\`\`\`bash
+# Switch to the target branch and update it
+git checkout ${target.branch}
+git pull origin/${target.branch}
+
+# Create the new backport branch
+git checkout -b fix/foo-${target.branch}
```
Comment (nickvergessen) [inline lib/pr.js:126]:
```suggestion
git checkout -b backport/${ there is a var for the PR number I guess}/${target.branch}
```

File: lib/pr.js [hunk 2]
```diff
@@ -115,7 +115,25 @@ module.exports = {
...
+
+# Cherry pick the change from the commit sha1 of the change against the default branch
+# This might cause conflicts. Resolve them.
+git cherry-pick abc123
```
Comment (nickvergessen) [inline lib/pr.js:130]: We could even list all the commit ids?

**Review by nickvergessen — COMMENTED** [review]
Still a nice improvement!

File: lib/pr.js (hunk 1 above)
Comment (ChristophWurst) [reply lib/pr.js:126]: tbh I was too lazy to get it to a 100%

File: lib/pr.js (hunk 2 above)
Comment (ChristophWurst) [reply lib/pr.js:130]: yes! could :P

File: lib/pr.js (hunk 1 above)
Comment (nickvergessen) [reply lib/pr.js:126]: All good, as said, already a huge improvement

**Pull request merged**